client/src/components/EventForm.js

```diff
@@ -131,24 +128,46 @@ export default function EventForm({ open }) {
                     value={newEvent.event_description}
                 />
 
-                <DefaultButton valid={valid} type='submit' buttonText='POST NEW EVENT' ></DefaultButton>
+                <EventTags
+                    eventtags={newEvent.eventtags}
+                    onCreateEventTag={addEventTag}
+                    onDeleteEventTag={deleteEventTag}
+                    onDeleteLastEventTag={deleteLastEventTag}>
+                </EventTags>
 
+                <DefaultButton valid={valid} type='submit' buttonText='POST NEW EVENT' ></DefaultButton>
             </FormWrapper>
+
+
+
         </>
     )
 }
 
+function createDaysOptions(numberOfDays = 31) {
+    let options = [];
+    for (let i = 1; i <= numberOfDays; i++) {
+        options.push(<option value={i}>{i}</option>);
+    }
+    return options;
+}
+
+
 const FormWrapper = styled.form`
 display:flex;
 flex-direction: column;
-margin: 1.2rem 2.3rem;
+align-content:left
+
 gap: 0.2rem;
 opacity: ${({ open, valid }) => open || valid ? '40%' : '100%'};
 border:solid 2px var(--mainorange);
 padding:10px;
 box-shadow: 0.2rem 0.3rem 0.3rem 0.5rem rgba(0,0,0, 35%);
-height:1rem
-width:1rem;
+height:100%;
+width:100%;
+margin-top:3rem;
+margin-left:1rem;
+margin-bottom:2rem;
```

Comment:
```suggestion
margin: 3rem 0 2rem 1rem;
```

Could be shortend to one property